lib/react_client/private_utils.dart

```diff
@@ -0,0 +1,12 @@
+@JS()
+library react_client.private_utils;
```

Comment:
I think this should be under `src`; we don't seem to use it in over_react or anything